src/distPlus/distribution/Mixed.re

```diff
@@ -61,29 +54,35 @@ module T =
       let truncatedDiscrete =
         Discrete.T.truncate(leftCutoff, rightCutoff, discrete);
 
-      make(~discrete=truncatedDiscrete, ~continuous=truncatedContinuous);
+      make(~discrete=truncatedDiscrete, ~continuous=truncatedContinuous, None, None);
     };
 
     let normalize = (t: t): t => {
+      let continuousIntegral = Continuous.T.Integral.get(t.continuous);
+      let discreteIntegral = Discrete.T.Integral.get(t.discrete);
```

Comment:
it could be worth making these and similar their own functions; i.e. law of demeter.